NEWS.md

```diff
@@ -1,5 +1,8 @@
 # forcats 0.2.0.9000
 
+# `fct_relabel()` now accepts objects coercible to functions 
+  by `rlang::as_function` (#91)
```

Comment:
Can you please add your github user name as described in http://style.tidyverse.org/news.html#during-development ?